src/Pages/hyunchanpark/Login/Login.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { withRouter } from 'react-router';
+import './Login.scss';
+
+const regObj = {
+  emailRegExp:
+    /[a-zA-Z0-9.-_+!]+@[a-zA-Z0-9]+\.[a-zA-Z0-9]{2,}(?:.[a-zA-Z0-9]{2,3})?/,
+  passwordRegExp: /[a-zA-Z0-9]{5,100}/,
+};
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { userId: '', userPw: '' };
+  }
+
+  validate = (value, regExp) => {
+    const reg = new RegExp(regExp);
+    let isValid = false;
+    if (reg.test(value)) isValid = true;
+    return isValid;
+  };
+
+  validateInputData = (id, pw) => {
+    return (
+      this.validate(id, regObj.emailRegExp) &&
+      this.validate(pw, regObj.passwordRegExp)
+    );
+  };
+
+  handleInput = e => {
+    this.setState(
+      {
+        [e.target.name]: e.target.value,
+      },
+      () => {
+        this.validateInputData(this.state.userId, this.state.userPw);
+      }
+    );
+  };
+
+  handleSubmit = e => {
+    e.preventDefault();
+
+    if (!this.validateInputData(this.state.userId, this.state.userPw)) return;
+
+    this.props.history.push('/main');
+  };
+
+  render() {
+    const { userId, userPw } = this.state;
+    const isDisabled = this.validateInputData(userId, userPw) ? false : true;
+
+    return (
+      <>
+        <section className="entire-container">
+          <div className="entire">
+            <div className="entire_top content">
+              <div className="login_container">
+                <h1>westagram</h1>
+                <div className="form_container">
+                  <form onSubmit={this.handleSubmit}>
+                    <input
+                      type="text"
+                      name="userId"
+                      placeholder="전화번호, 사용자 이름 또는 이메일"
+                      value={this.state.userId}
+                      onChange={this.handleInput}
+                    />
+                    <input
+                      type="password"
+                      name="userPw"
+                      placeholder="비밀번호"
+                      value={this.state.userPw}
+                      onChange={this.handleInput}
+                    />
+                    <Link to="/main">
+                      <button
+                        className="login-button"
+                        type="submit"
+                        disabled={isDisabled}
```

Comment:
```suggestion
                        disabled={!this.validateInputData(userId, userPw)}
```
위에 달아드린 코멘트와 이어지는 내용입니다!